### PR TITLE
cloud: Event page supports displaying more event types

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by srstack — APPROVED** [review]

File: tidb-cloud/tidb-cloud-events.md [hunk 1]
```diff
@@ -30,12 +30,21 @@ To view the events on the Events page, take the following steps:
...
+| ModifyClusterSize |   Events recorded the actions of modifying cluster size. | 
+| BackupCluster |   Events recorded the actions of backing up cluster. |  
+| RestoreFromCluster |   Events recorded the actions of restoring cluster. |  
+| CreateChangefeed |   Events recorded the actions of creating changefeed. |  
+| PauseChangefeed |   Events recorded the actions of pausing changefeed. | 
+| ResumeChangefeed |   Events recorded the actions of resuming changefeed. | 
+| DeleteChangefeed |   Events recorded the actions of deleting changefeed. |  
+| EditChangefeed |   Events recorded the actions of editing changefeed.|  
+| ScaleChangefeed |   Events recorded the actions of scaling changefeed. |  
+| FailedChangefeed |   Events recorded the failure of the changefeed. |
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:47]:
```suggestion
| CreateCluster |  Create a cluster |  
| PauseCluster |   Pause a cluster |  
| ResumeCluster |   Resume a cluster | 
| ModifyClusterSize |   Modify cluster size | 
| BackupCluster |   Back up a cluster |  
| RestoreFromCluster |   Restore a cluster |  
| CreateChangefeed |   Create a changefeed |  
| PauseChangefeed |   Pause a changefeed | 
| ResumeChangefeed |   Resume a changefeed | 
| DeleteChangefeed |   Delete a changefeed |  
| EditChangefeed |  Edit a changefeed |  
| ScaleChangefeed |   Scale the specification of a changefeed |  
| FailedChangefeed |   Changefeed failures |  
```

Comment (qiancai) [reply tidb-cloud/tidb-cloud-events.md:47]: Refer to https://docs.pingcap.com/tidbcloud/tidb-cloud-console-auditing#console-audit-event-types

File: tidb-cloud/tidb-cloud-events.md [hunk 2]
```diff
@@ -30,12 +30,21 @@ To view the events on the Events page, take the following steps:
 
 TiDB Cloud logs the following types of cluster events:
 
-| Event Type| Description |  Available since |
+| Event Type| Description | 
 |:--- |:--- |:--- |
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:34]:
```suggestion
|:--- |:--- |
```

**Review by ran-huang — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**